Source/WebCore/platform/audio/AudioUtilities.cpp

```diff
@@ -80,6 +80,8 @@ size_t timeToSampleFrame(double time, double sampleRate, SampleFrameRounding rou
 
 void applyNoise(float* values, size_t numberOfElementsToProcess, float standardDeviation)
 {
+    DisableMallocRestrictionsForCurrentThreadScope disableMallocRestrictions;
```

Comment:
Is this the right fix? Or should we be doing the allocations ahead of time somehow? Slowing down the audio thread in PB more than we need to just for the noise seems bad, no?

Reply:
Right, it'd be better if we could avoid heap allocations (didn't check if this was a viable option).

Reply:
The newest version of this patch should address this, mostly — while it'll be lazily allocated the first time, we'll keep reusing the same buffer for subsequent noise injection.